.vuepress/public/_redirects

```diff
@@ -1,84 +1,84 @@
-/guides/advanced_guides/concat.html                 /reference/under_the_hood/concat.html
-/guides/advanced_guides/bucket_sort.html            /reference/under_the_hood/bucket_sort.html
-/guides/advanced_guides/known_limitations.html      /reference/features/known_limitations.html
-/guides/advanced_guides/tokenization.html           /reference/under_the_hood/tokenization.html
-/references                                         /reference/api
+/guides/advanced_guides/field_properties.html       /learn/configuration/displayed_searchable_attributes.html
+/guides/advanced_guides/typotolerance.html          /learn/configuration/typo_tolerance.html
+/guides/advanced_guides/prefix.html                 /learn/advanced/prefix.html
+/guides/advanced_guides/concat.html                 /learn/advanced/concat.html
+/guides/advanced_guides/bucket_sort.html            /learn/core_concepts/relevancy.html#behavior
+/guides/advanced_guides/known_limitations.html      /learn/advanced/known_limitations.html
+/guides/advanced_guides/tokenization.html           /learn/advanced/tokenization.html
+/references                                         /reference/api/overview.html
 /references/indexes.html                            /reference/api/indexes.html
 /references/documents.html                          /reference/api/documents.html
 /references/search.html                             /reference/api/search.html
-/references/updates.html                            /reference/api/updates.html
+/references/updates.html                            /reference/api/tasks.html
 /references/keys.html                               /reference/api/keys.html
 /references/settings.html                           /reference/api/settings.html
-/references/synonyms.html                           /reference/api/synonyms.html
-/references/stop_words.html                         /reference/api/stop_words.html
-/references/ranking_rules.html                      /reference/api/ranking_rules.html
-/references/attributes_for_faceting.html            /reference/api/attributes_for_faceting.html
-/references/distinct_attribute.html                 /reference/api/distinct_attribute.html
-/references/searchable_attributes.html              /reference/api/searchable_attributes.html
-/references/displayed_attributes.html               /reference/api/displayed_attributes.html
+/references/synonyms.html                           /reference/api/settings.html
+/references/stop_words.html                         /reference/api/settings.html
+/references/ranking_rules.html                      /reference/api/settings.html
+/references/attributes_for_faceting.html            /reference/api/settings.html
+/references/distinct_attribute.html                 /reference/api/settings.html
+/references/searchable_attributes.html              /reference/api/settings.html
+/references/displayed_attributes.html               /reference/api/settings.html
```

Comment:
Should we redirect the reader to the specific heading instead of the settings page? That would be helpful